packages/manager/welcome/rollup.config.js

```diff
@@ -4,13 +4,20 @@ const config = rollupConfig({
   input: 'src/index.js',
 });
 
-export default [
-  config.cjs(),
-  config.umd({
-    output: {
-      globals: {
-        angular: 'angular',
+const outputs = [];
```

Comment:
```suggestion
const outputs = [config.cjs()];
```
In that way you can get rid of L9.